CHANGELOG.md

```diff
@@ -14,10 +14,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Improved how documentation is produced for VSCode compatibility (#485)
 - Enable access to builtin documentation parsed from `builtin.qnt` (#485)
 - The effect and mode checkers no longer complain about runs (#505)
+- Fixed missing lodash dependency (#484)
 
 ### Added
 
-- Fixed missing lodash dependency (#484)
 - A tutorial on integers (#484)
```

Comment:
This reference is wrong (noticed it in the PR description)

```suggestion
- A tutorial on integers (#495)
```

Reply:
thanks!